app/controllers/popularProductEnquiry.js

```diff
@@ -0,0 +1,30 @@
+const models = require("../models");
+const PopularProductEnquiry = models.PopularProductEnquiry;
+const consumeError = require("../functions/consumeError");
+
+module.exports = {
+  async index(req, res) {
+    try {
+      let getAllPopularProductEnquiry = await PopularProductEnquiry.findAll({
+        where: {},
+      });
+      return getAllPopularProductEnquiry;
+    } catch (error) {
+      consumeError(error);
+    }
+  },
+  async create(req, res) {
+    try {
+      const popularProductEnquiry = await PopularProductEnquiry.create({
+        name: req.body.name,
+        product_name: req.body.product_name,
```

Comment:
what if we are getting product_uuid in payload?

Reply:
where are you saving the user_uuid?

Reply:
where are we passing the status? @ChaitanyaV98

Reply:
1. @shubham all the popular products that we are showing are hard coded and we dont have uuid of the product @ShubhamJoshi-1994 .

Reply:
I have changed the code @ShubhamJoshi-1994, now we are able to add user_uuid and status as well.